src/modules/folder/folder.usecase.spec.ts

```diff
@@ -492,4 +493,32 @@ describe('FolderUseCases', () => {
       }
     });
   });
+
+  describe('get folder size', () => {
+    it('should return the folder size', async () => {
+      const mockSize = 123456789;
+
+      jest
+        .spyOn(folderRepository, 'calculateFolderSize')
+        .mockResolvedValueOnce(mockSize);
+
+      const result = await service.getFolderSizeByUuid('an-uuid');
+
+      expect(result).toBe(mockSize);
+      expect(folderRepository.calculateFolderSize).toHaveBeenCalledTimes(1);
+      expect(folderRepository.calculateFolderSize).toHaveBeenCalledWith(
+        'an-uuid',
+      );
+    });
+
+    it('should throw calculate folder size timeout exception', async () => {
+      jest
+        .spyOn(folderRepository, 'calculateFolderSize')
```

Comment:
Why not mocking the repo to return after more time than the timeout seconds? Then, you will be testing the abort behavior

Reply:
Stubbing the return and expecting to receive the error is testing the mocking library more than any other thing. I recommend to make the sequelize model last more than the timeout limit so the timeout will trigger itself